packages/synapse-react-client/src/components/AclEditor/useUpdateAcl.test.ts

```diff
@@ -165,5 +165,20 @@ describe('useUpdateAcl', () => {
         { principalId: MOCK_USER_ID_2, accessType: [ACCESS_TYPE.READ] },
       ]),
     )
+
+    // Verify that if we reset the dirty state, the list is re-sorted
+    act(() => {
+      result.current.resetDirtyState()
```

Comment:
We may want to reset the state and re-sort in various circumstances:
* After a save + close 
* After the user adds/removes a local ACL (the editor should reset and we should sort the inherited ACL)